lib/file_transfer_agent/remote_storage_util.js

```diff
@@ -292,11 +294,18 @@ function RemoteStorageUtil(connectionConfig) {
     let lastErr;
     const maxRetry = DEFAULT_MAX_RETRY;
 
+    Logger().info(
+      `Started downloading result. File: ${meta.dstFileName}, location: ${meta.stageInfo.location}`,
+    );
+    const timer = new ExecutionTimer().start();
     for (let retry = 0; retry < maxRetry; retry++) {
       // Download the file
       await utilClass.nativeDownloadFile(meta, fullDstPath, maxConcurrency);
```

Comment:
To ensure precise performance benchmarking and deterministic behavior, consider explicitly calling `timer.stop()` immediately after the download completes. This small addition makes the timing more accurate.